src/components/PrimaryPanes/Outline.js

```diff
@@ -162,7 +217,20 @@ export default connect(
     const selectedSource = getSelectedSource(state);
     return {
       symbols: getSymbols(state, selectedSource && selectedSource.toJS()),
-      selectedSource
+      selectedSource,
+      selectedLocation: getSelectedLocation(state),
+      getFunctionText: line =>
+        findFunctionText(
+          line,
+          selectedSource.toJS(),
+          getSymbols(state, selectedSource.toJS())
+        ),
+      getFunctionLocation: line =>
+        findClosestScope(getSymbols(state, selectedSource.toJS()).functions, {
```

Comment:
There is flow error 
```
Cannot get `getSymbols(...).functions` because property `functions` is missing in null or undefined
```
Should i check the existence of `getSymbols(...)` right here, return `undefined` from `getFunctionLocation` call and again check it in `onContextMenu` ?

Reply:
I'd recommend to save the result of `getSymbols` to a variable and check if it's not null before accessing `functions`.

Or if you like, you can fix `getSymbols` to always return `SymbolDeclarations`

Reply:
Turns out i don't need this helper at all